css/styles.css

```diff
@@ -0,0 +1,65 @@
+body {
+  background-color: #f0f0f0;
+}
+
+.image {
+  max-width: 100%;
+  border-radius: 5px;
+}
+
+header {
+  background-color: #552b81;
+  color: white;
+  width: 100%;
+}
+
+.head {
+  font-size: 24px;
+  font-weight: 400;
+}
+
+.container-main {
+  width: 50%;
+  margin: auto;
+}
```

Comment:
We already made a `.container` class for you guys to use in the `layout.css` file, you should just use that one so you don't have multiple tools for one situation